### ISSUE TITLE
Index not working

### ISSUE THREAD

**Issue by HenreyMine** [KorzhCom/EasyData#43]
When you use MetaEntityAttr Index it is work only in Edit view and didn't work on standart view.

**Comment by antifree** [comment]
Hello,

Do you mean that the index is not working in the grid?

**Comment by HenreyMine** [comment]
Yes. Columns stay in the same order. But when i click on Edit, i can see that Index work in that place.

**Comment by antifree** [comment]
Okay, we will fix it before the final release of 1.3.0

**Comment by antifree** [comment]
Done in version 1.3.0

**Comment by HenreyMine** [comment]
Thank you.
